File: maiuscolo o minuscolo/controllo.py
print("Inserisci un carattere")
c = input()
if c >= "a" and c <= "z":
    print("Carattere minuscolo")
else:
    if c >= "A" and c <= "Z":
        print("Carattere maiuscolo")
    else:
        print("Non è un carattere")

File: maiuscolo o minuscolo/controllo.java
import java.util.*;
import java.lang.Math;

public class JavaApplication {
    private static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {
        String c;
        
        System.out.println("Inserisci un carattere");
        c = input.nextLine();
        if (c.compareTo("a") >= 0 && c.compareTo("z") <= 0) {
            System.out.println("Carattere minuscolo");
        } else {
            if (c.compareTo("A") >= 0 && c.compareTo("Z") <= 0) {
                System.out.println("Carattere maiuscolo");
            } else {
                System.out.println("Non è un carattere");
            }
        }
    }
}
